src/tests/stdlib/contains_all.ab

```diff
@@ -0,0 +1,15 @@
+import { contains_all } from "std/text"
+
+// Output
```

Comment:
I suggested adapting the `contains_any` test because you're currently not testing all the edge cases, e.g. where the terms array is empty, or contains no matching terms.  Also, the other test labels its succeeding or failing test cases; you're just outputting an unqualified "Succeeded".